### PR TITLE
Fixes for dbproxy from client testing

### PR DESCRIPTION
**Pull request by jgd5**
Most of the fixes are related to passing ids and not found error handling.

### CODE REVIEW

File: go/dbproxy/dbproxy.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func Run(args []string) {
 	r.GET("/localtable/:name/:userid", localTableGetAll)
 	r.GET("/localtable/:name/:userid/:uniqueid", localTableGetOne)
 	r.DELETE("/localtable/:name/:userid/:uniqueid", localTableDeleteOne)
-	r.POST("/localtable/:name/:userid", localTableInsertOne)
+	r.POST("/localtable/:name/:userid/:uniqueid", localTableInsertOne)
```
Comment (gcampax) [inline go/dbproxy/dbproxy.go:46]: uniuqeids can be long and not URL safe. I think it's better to pass them in the POST body.

File: go/dbproxy/local_table.go [hunk 2]
```diff
@@ -110,7 +110,7 @@ func localTableDeleteOne(c *gin.Context) {
 		}
 		return
 	}
-	c.JSON(http.StatusOK, gin.H{"data": "ok"})
+	c.JSON(http.StatusOK, gin.H{"data": true})
```
Comment (gcampax) [inline go/dbproxy/local_table.go:113]: In other APIs, we use `{"result":"ok", "data": ...}`. It's best to be consistent I think.

File: go/dbproxy/dbproxy.go [hunk 3]
```diff
@@ -52,8 +56,8 @@ func Run(args []string) {
 	r.POST("/synctable/changes/:name/:userid", syncTableHandleChanges)
 	r.POST("/synctable/sync/:name/:userid/:millis", syncTableSyncAt)
 	r.POST("/synctable/replace/:name/:userid", syncTableReplaceAll)
-	r.POST("/synctable/:name/:userid/:millis", syncTableInsertIfRecent)
-	r.POST("/synctable/:name/:userid", syncTableInsertOne)
+	r.POST("/synctable/:name/:userid/uniqueid/:millis", syncTableInsertIfRecent)
```
Comment (gcampax) [inline go/dbproxy/dbproxy.go:59]: typo here? should be :uniqueid ?

**Review by gcampax — APPROVED** [review]
Ok

File: go/dbproxy/dbproxy.go (hunk 3 above)
Comment (jgd5) [reply go/dbproxy/dbproxy.go]: Fixed

**Pull request merged**